distribution/docs/src/main/resources/_contents/_installing/installing-intro-contents.adoc

```diff
@@ -0,0 +1,357 @@
+
+[IMPORTANT]
+====
+Although ${branding} can be installed by any user, it is recommended for security reasons to have a non-`root` user execute the ${branding} installation.
+====
+
+=== ${branding} Installation Prerequisites
+
+These are the system/environment requirements to configure _prior_ to an installation.
+
+==== Directory Permissions
```

Comment:
This section assumes that the `etc` and `data` directories already exist. However, those will not be created until section 2.2. Should we move this section after 2.2? Or should this be part of the security section and how to secure DDF?